cli/azd/pkg/osutil/osversion/osversion.go

```diff
@@ -2,6 +2,8 @@
 
 package osversion
 
-func GetVersion() string {
-	return ""
+import "errors"
+
+func GetVersion() (string, error) {
+	return "", errors.New("unsupported OS")
```

Comment:
If you expand the context, you'll note this version is excluded by build flags for all the platforms we care about.

Reply:
LGTM